test/libsolidity/syntaxTests/dataLocations/function_return_parameters_with_data_location_fine.sol

```diff
@@ -4,3 +4,6 @@ contract C {
     function h() public pure returns(uint[] memory) {}
     function i() external pure returns(uint[] memory) {}
 }
+// ----
+// Warning 6321: (51-64): Unnamed return parameter will never be set.
+// Warning 6321: (134-147): Unnamed return parameter will never be set.
```

Comment:
Do we want these? This goes against https://github.com/ethereum/solidity/issues/4751#issuecomment-597781232 at least...

Reply:
Ah nevermind, it's the first two functions, right?

Reply:
Yep, the first two, correct